app/src/main/java/com/example/caregiver/Login.java

```diff
@@ -74,10 +74,10 @@ private void loginFirebaseCall(String email, String password){
      * Navigates to Dashboard after successful sign in through Firebase
      */
     private void navigateToDashboard(String userId){
-        DatabaseReference ref = FirebaseDatabase.getInstance().getReference().child("users/" + userId);
         SharedPreferences preferences = PreferenceManager.getDefaultSharedPreferences(this);
         SharedPreferences.Editor editor = preferences.edit();
-        editor.putString("userId", userId);
+        DatabaseReference database = FirebaseDatabase.getInstance().getReference();
+        DatabaseReference ref = database.child("users/" + userId);
```

Comment:
I'm curious, is there a reason for splitting this into 2 lines?

Reply:
Oh nah no reason. I just copy + paste the code fragment from another page. I could leave it back to one line since technically we don't really need to reuse the database variable. Good point!